multinode-demo/fullnode.sh

```diff
@@ -72,32 +72,51 @@ rsync_url() { # adds the 'rsync://` prefix to URLs that need it
   echo "rsync://$url"
 }
 
-setup_vote_account() {
+setup_vote_and_stake_accounts() {
   declare entrypoint_ip=$1
   declare node_keypair_path=$2
   declare vote_keypair_path=$3
-  declare stake=$4
+  declare stake_keypair_path=$4
+  declare stake=$5
 
-  declare node_keypair
-  node_keypair=$($solana_wallet --keypair "$node_keypair_path" address)
+  declare node_id
+  node_id=$($solana_wallet --keypair "$node_keypair_path" address)
 
-  declare vote_keypair
-  vote_keypair=$($solana_wallet --keypair "$vote_keypair_path" address)
+  declare vote_id
+  vote_id=$($solana_wallet --keypair "$vote_keypair_path" address)
 
-  if [[ -f "$vote_keypair_path".configured ]]; then
-    echo "Vote account has already been configured"
+  declare stake_id
```

Comment:
`stake_keypair`? (see above)

Reply:
> r+, with one question in `fullnode.sh`
> Thanks for the `App` changes in wallet. `wallet.rs` is getting pretty unwieldy; this PR makes me consider several ways to reorganize, but no need to hold up your functionality

yeah: there were a bunch of deltas between the two builder stacks